frameworks/GAMA/exec.py

```diff
@@ -3,13 +3,12 @@
 import sys
 import tempfile as tmp
 
+os.environ['JOBLIB_TEMP_FOLDER'] = tmp.gettempdir()
 if sys.platform == 'darwin':
     os.environ['OBJC_DISABLE_INITIALIZE_FORK_SAFETY'] = 'YES'
-os.environ['JOBLIB_TEMP_FOLDER'] = tmp.gettempdir()
-os.environ['OMP_NUM_THREADS'] = '1'
-os.environ['OPENBLAS_NUM_THREADS'] = '1'
-os.environ['MKL_NUM_THREADS'] = '1'
-
+    os.environ['OMP_NUM_THREADS'] = '1'
+    os.environ['OPENBLAS_NUM_THREADS'] = '1'
+    os.environ['MKL_NUM_THREADS'] = '1'
```

Comment:
@PGijsbers need approval from author

Reply:
For similar reasons as auto-sklearn, I think to guarantee no accidental violation of cpu constraints would need to be used for GAMA.